db/migrate/20220219083323_add_users_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] I also replaced line 3 from `add_reference :comments, :user, null: false, foreign_key: true` to `add_reference :comments, :author, foreign_key: { to_table: :users }`